http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -423,7 +423,7 @@ class ClusterMonitorActor(
     for {
       dbCluster <- getDbCluster
       next <- dbCluster.status match {
-        case status if status.isMonitored =>
+        case status if status.isMonitored && dbCluster.dataprocInfo.isDefined =>
```

Comment:
Perhaps the check could be here instead, to prevent the cluster from even being monitored?

https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSupervisor.scala#L325

Reply:
ah cool 👍 updated!